src/OpenLoco/Title.cpp

```diff
@@ -3,16 +3,90 @@
 #include "CompanyManager.h"
 #include "Gui.h"
 #include "Interop/Interop.hpp"
+#include "Map/TileManager.h"
 #include "OpenLoco.h"
 #include "Scenario.h"
 #include "Ui/WindowManager.h"
 
+#include <variant>
+#include <vector>
+
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Title
 {
     static loco_global<uint8_t, 0x00508F1A> _gameSpeed;
 
+    struct WaitStep
+    {
+        uint16_t duration;
+    };
+
+    struct ReloadStep
+    {
+        uint16_t arg;
+    };
+
+    struct MoveStep
+    {
+        coord_t xPos;
```

Comment:
Why not just make this a tile position or a map position?

Reply:
Mostly because I wanted to keep the initialisation simple. I'd like a TitleSequence array to read like a script. Particularly, here:
https://github.com/OpenLoco/OpenLoco/blob/2d4af4a26bfab7539b856486553a761de615ae54/src/OpenLoco/Title.cpp#L59
I suppose we could create an extra constructor that takes these two coordinates and stores them as a map_pos, though. That saves the hassle of doing it later on here:
https://github.com/OpenLoco/OpenLoco/blob/2d4af4a26bfab7539b856486553a761de615ae54/src/OpenLoco/Title.cpp#L207
Is that what you had in mind as well?

Reply:
Yes, you can just make MoveStep and alias of a tile position then you don't need to make a constructor

Reply:
Nice idea! Done